dataregistry-extension/src/csvviewer/index.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license BSD-3-Clause
+ *
+ * Copyright (c) 2019 Project Jupyter Contributors.
+ * Distributed under the terms of the 3-Clause BSD License.
+ */
+
+import { JupyterFrontEndPlugin } from "@jupyterlab/application";
+import { IRegistry } from "./../registry";
+import { activate } from "./activate";
+
```

Comment:
I think this should also export all from the datatypes file, so that other modules can refer to this datatypes